IDE/Renesas/e2studio/RA6M4/test/src/test_main.c

```diff
@@ -381,22 +396,26 @@ void sce_test(void)
 
     #else
 
+    int TCP_connect_retry = 0;
     wolfSSL_TLS_client_init();
 
     do {
 
-          info[i].port = DEFAULT_PORT;
-          info[i].cipher = cipherlist[i];
-          info[i].ctx = client_ctx;
-          info[i].id = i;
-
-          memset(info[i].name, 0, sizeof(info[i].name));
-          sprintf(info[i].name, "wolfSSL_TLS_client_do(%02d)", i);
+        info[i].port = DEFAULT_PORT;
+        info[i].cipher = cipherlist[i];
+        info[i].ctx = client_ctx;
+        info[i].id = i;
 
-          wolfSSL_TLS_client_do(&info[i]);
+        memset(info[i].name, 0, sizeof(info[i].name));
```

Comment:
Should these calls be changed to our abstracted `XMEMSET` and `XSPRINTF`?

Reply:
Agree, thanks, Fixed.